tests/test_lint.py

```diff
@@ -185,62 +184,84 @@ def test_assert_model_parameters_in_condition_or_parameter_table():
 
 
 def test_assert_noise_distributions_valid():
-    measurement_df = pd.DataFrame(data={
-        'observableId': ['0obsPar1noisePar',
-                         '2obsPar0noisePar'],
-        'simulationConditionId': ['condition1', 'condition1'],
-        'preequilibrationConditionId': ['', ''],
-        'time': [1.0, 2.0],
-        'measurement': [1.0, 2.0],
-        'observableParameters': ['', ''],
-        'noiseParameters': ['', ''],
-        'noiseDistribution': ['', ''],
+    observable_df = pd.DataFrame(data={
+        OBSERVABLE_ID: ['0obsPar1noisePar',
+                        '2obsPar0noisePar'],
+        NOISE_PARAMETERS: ['', ''],
+        NOISE_DISTRIBUTION: ['', ''],
```

Comment:
also test with that column non-existent?

Reply:
done